backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/createFrom/po_from_so/impl/CreatePOFromSOsAggregator.java

```diff
@@ -96,6 +102,7 @@ protected I_C_Order createGroup(final Object vendorBPartnerValue, final I_C_Orde
 	{
 		if (CreatePOFromSOsAggregationKeyBuilder.KEY_SKIP.equals(vendorBPartnerValue))
 		{
+			skippedSalesOrderLineIds.add(OrderLineId.ofRepoId(salesOrderLine.getC_OrderLine_ID()));
```

Comment:
please log this to `Loggables.withLogger(logger, Level.DEBUG)` (also add a static logger)

PS: ah, no..you are already doing that in `CreatePOFromSOsAggregationKeyBuilder.java`, right?

Reply:
Logged in CreatePOFromSOsAggregator#addItemToGroup. We're not logging in CreatePOFromSOsAggregationKeyBuilder.java